src/sinks/gcp/chronicle_unstructured.rs

```diff
@@ -210,6 +226,18 @@ pub enum ChronicleError {
 #[typetag::serde(name = "gcp_chronicle_unstructured")]
 impl SinkConfig for ChronicleUnstructuredConfig {
     async fn build(&self, cx: SinkContext) -> crate::Result<(VectorSink, Healthcheck)> {
+        if let Some(labels) = self.labels.clone() {
```

Comment:
```suggestion
        if let Some(ref labels) = self.labels {
```

Reply:
Removed this code since ingest labels don't have the same requirements as RBAC labels.